pom.xml

```diff
@@ -169,6 +198,28 @@
                     <plugin>
                         <groupId>org.codehaus.mojo</groupId>
                         <artifactId>rpm-maven-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>attach-rpm</id>
+                                <goals>
+                                    <goal>attached-rpm</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                        <configuration>
+                            <group>Applications/Archiving</group>
+                            <mappings combine.children="append">
+                                <mapping>
+                                    <directory>/opt/${dans-provider-name}/${project.artifactId}/lic</directory>
+                                    <filemode>755</filemode>
```

Comment:
@vesaakerman : why filemode 755?

Reply:
I have obviously copy-pasted it.  Indeed not necessary, the parent pom default 744 is good here.